VSTS/SonarQube/Tasks/ScannerCli/scanner-cli-task.ts

```diff
@@ -12,7 +13,9 @@ async function run(): Promise<void> {
     try {
         SonarQubeHelper.exitOnPrBuild();
 
-        var sonarScannerToolPath: string = path.join(__dirname, "sonar-scanner/bin/sonar-scanner.bat");
+        var isWindows = os.type().match(/^Win/);
+        var scannerScriptExtension: string = isWindows ? ".bat" : "";
+        var sonarScannerToolPath: string = path.join(__dirname, "sonar-scanner/bin/sonar-scanner", scannerScriptExtension);
```

Comment:
I think this:
"sonar-scanner/bin/sonar-scanner", scannerScriptExtension

should be changed to:
"sonar-scanner/bin/sonar-scanner" + scannerScriptExtension